blank_project/gulpfile.js

```diff
@@ -1,30 +1,20 @@
 const gulp = require("gulp");
 const nearUtils = require("near-shell/gulp-utils");
 
-gulp.task("build:model", callback => {
-  nearUtils.generateBindings("model.ts", "../out/model.near.ts", callback);
-});
 
-gulp.task("build:bindings", ["build:model"], callback => {
-  nearUtils.generateBindings("main.ts", "../out/main.near.ts", callback);
-});
+function build_bindings(done){
```

Comment:
just a small thing, but let's keep function names consistent - generateBindingsMain and generateBindingsModel?